src/js/main.js

```diff
@@ -202,12 +202,18 @@ const sticker = {
 stickable.onmousedown = e => {
 	if (e.which !== 1) return; // Only work on left mouse button
 	const onSticker = e.target.classList.contains("sticker");
-	if (onSticker && !sticker.current) {
+	if (onSticker) {
+		const offsetLeft = parseInt(e.target.style.getPropertyValue("--x"), 10);
+		const offsetTop = parseInt(e.target.style.getPropertyValue("--y"), 10);
+		sticker.offsetX = mouse.x - offsetLeft;
+		sticker.offsetY = mouse.y - offsetTop;
 		// Move clicked sticker
 		sticker.current = e.target;
 		sticker.current.classList.add("dragging");
 	} else {
 		// Create new sticker
+		sticker.offsetX = 0;
```

Comment:
Also, when generating a sticker, there is no offset set, which will result in `NaN` when the updateSticker function runs.